### DISCUSSION TITLE
Is it possible to download .bin over internet and flash esp with it?

### DISCUSSION THREAD

**Discussion by shoomilka** [esp8266/Arduino#8276] [Q&A] [answered]
Hi,

I want esp to download .bin from my hosting over URL and to update itself. Is it possible? Cannot find any about this.
I know my washing machine doing so.

Best,
Uliana

**Comment by JAndrassy** [comment, marked as answer]
see the ESP8266httpUpdate library

**Reply by shoomilka** [reply]
Thank you very much!!!
